config/app.config

```diff
@@ -45,6 +45,10 @@ irsa.gator.hostname     = "https://irsa.ipac.caltech.edu"
 most.host = "https://irsasearchops.ipac.caltech.edu/cgi-bin/MOST/nph-most"
 // IRSA Periodogram API
 irsa.gator.service.periodogram.url = "https://irsa.ipac.caltech.edu/cgi-bin/periodogram/nph-periodogram_api"
+// IRSA hips map access: from "file" or from "url" && hips url
+irsa.hips.list.source="url"
+irsa.hips.masterUrl="https://irsadev.ipac.caltech.edu/data/hips/list"
+irsa.hips.url.base="irsa.ipac.caltech.edu"
```

Comment:
I noticed protocol is not included.  What is the logic for choosing the protocol?
I find it's more straight forward to use either relative or full url, including protocol and ports as the base

Reply:
Maybe there is another way to do this instead of passing build properties. The three of us should talk about that.